### PR TITLE
Add Ideal Size node

### PR DESCRIPTION
**Pull request by JPPhoto**
## What type of PR is this? (check all applicable)

- [ ] Refactor
- [X] Feature
- [ ] Bug Fix
- [ ] Optimization
- [ ] Documentation Update
- [ ] Community Node Submission

## Have you discussed this change with the InvokeAI team?
- [X] Yes
- [ ] No, because:

      
## Have you updated all relevant documentation?
- [X] Yes
- [ ] No

## Description
The Ideal Size node is useful for High-Res Optimization as it gives the optimum size for creating an initial generation with minimal artifacts (duplication and other strangeness) from today's models.

After inclusion, front end graph generation can be simplified by offloading calculations for HRO initial generation to this node.

## Related Tickets & Documents

- Related Issue #
- Closes #

## Added/updated tests?

- [ ] Yes
- [ ] No : _please replace this line with details on why tests
      have not been included_

### CODE REVIEW

File: invokeai/app/invocations/latent.py
Comment (psychedelicious) [inline invokeai/app/invocations/latent.py:1254]: Can these descriptions be a bit more detailed? "Dimensional multiplier" in particular does not tell me anything without reading the code (and even then the intention isn't clear)

Comment (JPPhoto) [reply invokeai/app/invocations/latent.py]: Just updated them, hope they work.

**Review by psychedelicious — APPROVED** [review]

**Comment by psychedelicious** [comment]
Needs codeowner review from @Millu @hipsterusername

**Review by Millu — APPROVED** [review]

**Pull request merged**
